src/libraries/SignedCallsLib.sol

```diff
@@ -27,7 +28,8 @@ library SignedCallsLib {
                 signedCalls.calls.hash(),
                 signedCalls.nonce,
                 signedCalls.keyHash,
-                signedCalls.shouldRevert
+                signedCalls.shouldRevert,
+                keccak256(signedCalls.hookData)
```

Comment:
I was thinking that there would be a separate type with hook data, thoughts? Worst case someone could sign over empty data but maybe worth just having another type?

Reply:
Removed hookData in this PR from offline convo